### ISSUE TITLE
Descriptions take a long time when using VO

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#528]
https://github.com/phetsims/qa/issues/689

I noticed that when I move the balloons and which between them very quickly, it takes a while for VO to catch up with the descriptions. When I pause, it can take several seconds for VO to stop going through all of the events that took place. Once you watch the video, you will see what I am referring to. 
This is much different from when I was testing with John Travoltage. I'm sure if a different design decision was made or if this is a bug. @terracoda please let me know. 

https://drive.google.com/file/d/16z2kEXX-paIGYy_qTov_LUPA8C03fW2E/view?usp=sharing

**Comment by terracoda** [comment]
The description responses are more complex and longer with BASE, that is for sure.

Keep in mind that blind learners will likely have their screen readers at a much faster rate than the default rate. This greatly reduces how much the screen reader gets behind.

With 2 balloons, some description pile-ups are expected.

Your example in the video does indeed not sound optimal, but interaction is not likely something a blind learner will do. 

@jessegreenberg and I have done some optimization in the responses. We are satisfied that the behaviour is acceptable, and we agree that more optimization would be ideal. When we publish this sim with full mobile accessibility, we will be able to continue the optimization of the responses.

Thanks for you comment @Nancy-Salpepi, but we won't be able to do further optimization for this release. It will happen for the next release. Marking as deferred.

**Comment by jessegreenberg** [comment]
Based on the comment above, I am going to close this issue. I discussed with @terracoda on slack and we will keep an eye out for this issue as we redeploy the sim for #583 . But we also recognize that a lot of thought went into this sim and its current behavior. Its a balance between taking too long and hearing all the important events.
